unreleased_history/new_features/attribute_group_support.md

```diff
@@ -0,0 +1 @@
+Introduce AttributeGroup by adding the first AttributeGroup support APIs, MultiGetEntity() and PutEntity(). Through the use of Column Families, AttributeGroup enables users to logically group wide-column entities. More APIs to support AttributeGroup will come soon, including GetEntity, Iterators and others.
```

Comment:
We should probably have a dedicated entry for `PutEntity`, especially since `MultiGetEntity` made the 8.8 cutoff.